pom.xml

```diff
@@ -131,7 +131,7 @@
         <dependency>
             <groupId>com.sun.xml.bind</groupId>
             <artifactId>jaxb-core</artifactId>
-            <version>${java.ee.extras}</version>
+            <version>2.3.0.1</version>
```

Comment:
Wow that's sketchy that there was [no 2.3.1 release](https://mvnrepository.com/artifact/com.sun.xml.bind/jaxb-core) of this versus the other releases (api and impl).